apps/web/components/index.ts

```diff
@@ -1,50 +1,9 @@
-export * from './Card/Card'
-export * from './Categories/Categories'
-export * from './Header/Header'
-export * from './Sidebar/Sidebar'
-export * from './SearchInput/SearchInput'
-export * from './LanguageToggler/LanguageToggler'
-export * from './Sticky/Sticky'
-export * from './Heading/Heading'
-export * from './LinkCardList/LinkCardList'
-export * from './Timeline/TimelineSection'
-export * from './SideMenu/SideMenu'
-export * from './EmailSignup/EmailSignup'
-export * from './StoryList/StoryList'
-export * from './Content/Content'
-export * from './ContentLink/ContentLink'
-export * from './LatestNewsSection'
-export * from './NewsCard'
-export * from './LogoList/LogoList'
-export * from './List/List'
-export * from './AboutLatestNews/AboutLatestNews'
-export * from './Bullet/Bullet'
-export * from './BackgroundImage/BackgroundImage'
-export * from './BulletList/BulletList'
-export * from './SkipToMainContent/SkipToMainContent'
-export * from './PageLoader/PageLoader'
-export * from './FilteredCards/FilteredCards'
-export * from './FixedNav/FixedNav'
-export * from './SidebarBox/SidebarBox'
-export * from './SidebarSubNav/SidebarSubNav'
-export * from './RichText/RichText'
-export * from './SidebarNavigation/SidebarNavigation'
-export * from './LifeEventsCardsSection/LifeEventsCardsSection'
-export * from './IntroductionSection/IntroductionSection'
-export * from './Section/Section'
-export * from './FrontpageSlider/FrontpageSlider'
-export * from './EmailSignup/EmailSignup'
-export * from './HeadWithSocialSharing/HeadWithSocialSharing'
-export * from './TellUsAStory/TellUsAStory'
-export * from './ContactUs/ContactUs'
-export * from './ChatPanel'
-export * from './AnchorNavigation/AnchorNavigation'
-export * from './Main/Main'
-export * from './Select/Select'
-export * from './SubpageMainContent'
-export * from './SubpageDetailsContent'
-export * from './ServiceInformation'
-export * from './ServiceList'
-export * from './OpenApiView'
-export * from './OpenApiDocumentation'
-export * from './InstitutionPanel/InstitutionPanel'
+// Required ES6 syntax for TypeScript isolatedModules.
+export * from './real'
+
+// This module should never be imported for performance reasons. Instead
+// import components from `@island.is/web/components`. Read
+// `libs/shared/babel/README.md` for more details.
+throw new Error(
+  'Do not import web components from components/index.ts. Use `@island.is/web/components` instead.',
+)
```

Comment:
Did you only add the `reel` file to be able to throw the error on relative imports?

Reply:
Yeah. I couldn't find another way (read: an eslint rule) to block relative imports of this module. Though it might be easy to write from scratch.

Reply:
Yeah, an eslint rule would be optimal :) ... I can see someone importing straight from real 🙈

Reply:
Yeah. I made sure that index.ts exports all the same stuff.

That makes most IDE's suggest/autocomplete either "@island.is/web/components" (good!) or "../components" (bad but caught immediately and hopefully fixed correctly). Third IDE suggestion is "../components/real" >_<

Thought about naming the file DO_NOT_IMPORT.ts 😝

Oh well, might be fun refactor week task to write a custom eslint rule.

Reply:
haha `DO_NOT_IMPORT.ts` is actually pretty good!

Reply:
Doesn't this work if you use [`no-restricted-imports`](https://github.com/island-is/island.is/blob/main/.eslintrc#L46)  and pass the relative path to `../components/*`?